ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -941,6 +941,72 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-advanced
+- as: aws-rosa-sts-advanced-f360-content
+  cron: 9 21 2,9,16,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_SECURITY_GROUP: "true"
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.15"
+    test:
+    - as: test-ocp4-pci-dss-node
+      cli: latest
+      commands: |
+        export PROFILE=pci-dss-node
+        export PRODUCT=ocp4
+        export CONTENT_IMAGE=ghcr.io/complianceascode/k8scontent:latest
+        export component=ocp4-content-ds
```

Comment:
Once https://github.com/ComplianceAsCode/ocp4e2e/pull/37 lands we can add a flag to bypass remediation workflow when testing managed offerings.

```
export BYPASS_REMEDIATIONS=true
```

Reply:
This should also make it safer to reuse the same ROSA environment for all Compliance Operator testing since we're not making changes to the config that would affect subsequent tests.